test/integration/dockerregistryclient_test.go

```diff
@@ -16,9 +16,9 @@ func TestRegistryClientConnect(t *testing.T) {
 		t.Fatal(err)
 	}
 	for _, s := range []string{"index.docker.io", "https://docker.io", "https://index.docker.io"} {
-		otherConn, err := c.Connect(s, false)
-		if err != nil {
-			t.Errorf("%s: can't connect: %v", s, err)
+		otherConn, connErr := c.Connect(s, false, true)
```

Comment:
I think I'm misunderstanding how this is compiling with that third `true` argument to `Connect`: https://github.com/openshift/origin/blob/master/pkg/dockerregistry/client.go#L39